cime_config/config_grids.xml

```diff
@@ -2683,6 +2718,13 @@
       <desc>SOwISC12to60E2r4 is a MPAS ice/ocean grid with enhanced resolution of 12km in the Southern Ocean around Antarctica. The high resolution regions smoothly transition to the background resolution of the standard low resolution 60to30km grid:</desc>
     </domain>
 
+    <domain name="ECwISC30to60E2r1">
+      <nx>237984</nx>
+      <ny>1</ny>
+      <file grid="ice|ocn">$DIN_LOC_ROOT/share/domains/domain.ocn.ECwISC30to60E2r1.201007.nc</file>
+      <desc>ECwISC30to60E2r1 is a MPAS ocean grid generated with the jigsaw/compass process using the eddy closure density function that has 30 km gridcells at the equator, 60 km at mid-latitudes, and 35 km at high latitudes. Additionally, it has ocean under landice cavities:</desc>
```

Comment:
```suggestion
      <desc>ECwISC30to60E2r1 is a MPAS ocean grid generated with the jigsaw/compass process using the eddy closure density function that has 30 km gridcells at the equator, 60 km at mid-latitudes, and 35 km at high latitudes. Additionally, it has ocean in ice-shelf cavities:</desc>
```
I don't think "landice cavities" are really a thing.  And The ocean is in the cavities, under ice shelves so it either needs to be "in ice-shelf cavities" or "under ice shelves".

Reply:
Another 'copy from elsewhere' job - thanks for noticing, I've made the suggested change.

Reply:
Ah, I see!  I'll resist the urge to use the "blame" GitHub feature...